modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js

```diff
@@ -147,23 +161,38 @@ export default function UpperToolbar({displayNames, languageIds, version}) {
 
 	const saveDefinition = () => {
 		const successMessage = Liferay.Language.get('workflow-saved');
+		const duplicatedAlert = Liferay.Language.get(
```

Comment:
@MariMalta I think it would be better to add the word 'Message' in the end of the name.